Source/WebCore/dom/TemplateContentDocumentFragment.h

```diff
@@ -34,24 +34,24 @@ namespace WebCore {
 class TemplateContentDocumentFragment final : public DocumentFragment {
     WTF_MAKE_ISO_ALLOCATED(TemplateContentDocumentFragment);
 public:
-    static Ref<TemplateContentDocumentFragment> create(Document& document, const Element* host)
+    static Ref<TemplateContentDocumentFragment> create(Document& document, Element& host)
     {
         return adoptRef(*new TemplateContentDocumentFragment(document, host));
     }
 
-    const Element* host() const { return m_host; }
+    const Element* host() const { return m_host.get(); }
```

Comment:
Given that this return a `const Element*`. We might be able to use a `WeakPtr<const Element>` for the data member, `const Element&` for the constructor parameter, thus avoiding the const casting at the call sites.

Reply:
Yeah, we could. I was thinking of that but const here isn't all that useful. I'll go do that anyway.